src/Auth/ControllerAuthorize.php

```diff
@@ -50,7 +50,7 @@ class ControllerAuthorize extends BaseAuthorize
     protected $_Controller;
 
     /**
-     * {@inheritDoc}
+     * @inheritDoc
```

Comment:
This change makes our codesniffer unhappy. So the related sniff will also have to be updated.

Reply:
Done in https://github.com/cakephp/cakephp-codesniffer/pull/215